tests/testleopard.nim

```diff
@@ -98,8 +99,8 @@ suite "Leopard simple Encode/Decode":
     recovered = newSeq[seq[byte]](DataCount)
 
   teardown:
-    encoder.free()
-    decoder.free()
+    discard #encoder.free()
+    #decoder.free()
```

Comment:
I'm going to uncomment this once I'm going to fix the issue with error: `SIGILL: Illegal operation.` when `free()` is getting called.